libs/testserver/server.go

```diff
@@ -29,6 +42,10 @@ func New(t testutil.TestingT) *Server {
 
 type HandlerFunc func(req *http.Request) (resp any, err error)
 
+func (s *Server) Requests() []Request {
+	return s.requests
```

Comment:
This Getter pattern adds unnecessary noise, could you make Requests exportable and access it directly?

Reply:
done